nix/outputs.nix

```diff
@@ -0,0 +1,64 @@
+{ inputs, system }:
+
+let
+  inherit (pkgs) lib;
+
+  pkgs = import ./pkgs.nix { inherit inputs system; };
+
+  utils = import ./utils.nix { inherit pkgs lib; };
+
+  project = import ./project.nix { inherit inputs pkgs lib; };
+
+  mkShell = ghc: import ./shell.nix { inherit inputs pkgs lib project utils ghc; };
+
+  exposed-haskell-packages = {
+    kes-agent-test = project.flake'.packages."kes-agent:test:kes-agent-tests";
+    kes-agent = project.flake'.packages."kes-agent:exe:kes-agent";
+    kes-service-client-demo = project.flake'.packages."kes-agent:exe:kes-service-client-demo";
+    kes-agent-control = project.flake'.packages."kes-agent:exe:kes-agent-control";
+  };
+
+  static-haskell-packages = {
+    # TODO: the pkgs.gitMinimal (needed for the kes-agent version command) at compile time gives an error for the musl64 build
```

Comment:
@zeme-wana, is there a fix or trick for this?

Reply:
Unfortunately there is not :/ There is an alternative that Plutus uses; see [here](https://github.com/IntersectMBO/plutus/blob/2aca7057cc619a4557206ef630aee9df7ebd4c74/nix/project.nix#L61) and [here](https://github.com/IntersectMBO/plutus/blob/master/plutus-core/plutus-core/src/Data/Version/Extras.hs).